public/scripts/iobserver.js

```diff
@@ -3,43 +3,48 @@
  * Lazy loading of images using an intersection observer.
  */
 
-export default (document) => {
+export default function iObserver(document) {
```

Comment:
Just to be enough explicit, let's call this `intersectionObserver`...
```suggestion
export default function intersectionObserver(document) {
```